src/Bicep.Cli/Services/TemplateEvaluation.cs

```diff
@@ -18,14 +18,26 @@
 
 namespace Bicep.Cli.Services
 {
-    public class TemplateEvaluatorService
+    public class TemplateEvaluation
     {
         private const string DummyTenantId = "";
         private const string DummyManagementGroupName = "";
         private const string DummySubscriptionId = "";
         private const string DummyResourceGroupName = "";
         private const string DummyLocation = "";
         private static readonly Regex templateSchemaPattern = new Regex(@"https?://schema\.management\.azure\.com/schemas/[0-9a-zA-Z-]+/(?<templateType>[a-zA-Z]+)Template\.json#?", RegexOptions.Compiled);
```

Comment:
Sorry I missed this in the previous PR. We should consider using regex source generator: https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-source-generators to replace the static field.